kiwi/tasks/system_build.py

```diff
@@ -190,7 +190,7 @@ def process(self):
 
         # setup permanent image repositories after cleanup
         repositories = self.xml_state.get_repository_sections()
-        if bool(next(xr for xr in repositories if xr.get_imageinclude())):
+        if bool(list(xr for xr in repositories if xr.get_imageinclude())):
```

Comment:
This is not entirely correct. Your previous code was to break the loop as soon as there is anything. So I rewrote this into oneliner. Your version says "if there is built list that contains some elements". I.e. you loop until the very end. Also `next` returns only an iterator while `list` builds that fully in the memory. If you want go that way, than you could just:
```python
if [x for x in repositories if x.get_imageinclude()]:
   ...
```
But this is just not as efficient as my original example.